src/main/java/uk/gov/digital/ho/proving/income/IpsSearchPage.java

```diff
@@ -98,4 +113,30 @@ public WebElement getPageHeading() {
         wait.until(ExpectedConditions.presenceOfElementLocated(By.id("pageDynamicHeading")));
         return pageHeading;
     }
+
+    public WebElement getPageHeadingContent() {
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("outcomeBoxSummary")));
+        return pageHeadingContent;
+    }
+
+    public WebElement getResultHeading() {
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("div[ng-if=haveResult] > h2")));
+        return resultHeading;
+    }
+
+    public WebElement getApplicantFullName() {
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("yourSearchIndividualName0")));
+        return applicantFullName;
+    }
+
+    public WebElement getIncomeFromDate() {
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("outcomeFromDate0")));
+        return incomeFromDate;
+    }
+
+    public WebElement getIncomeToDate() {
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("outcomeToDate0")));
```

Comment:
For this and the methods directly above, those IDs have already been defined.  Is there any way we can re-use the previous definition so we can DRY?